testing/telegram-tdlib/APKBUILD

```diff
@@ -0,0 +1,32 @@
+# Contributor: Oleg Titov <oleg.titov@gmail.com>
+# Maintainer: Oleg Titov <oleg.titov@gmail.com>
+pkgname=telegram-tdlib
+pkgver=1.4.0
+pkgrel=0
+pkgdesc="Cross-platform library for building Telegram clients"
+url="https://core.telegram.org/tdlib"
+arch="all"
+license="BSL-1.0"
+options="!check" # No test suite from upstream
+depends=""
+makedepends="cmake openssl-dev zlib-dev gperf linux-headers"
+subpackages="$pkgname-static $pkgname-dev"
+source="$pkgname-$pkgver.tar.gz::https://github.com/tdlib/td/archive/v$pkgver.tar.gz"
+builddir="$srcdir/td-$pkgver"
+
+build() {
+	mkdir build
+	cd build
+	cmake -DCMAKE_BUILD_TYPE=Release \
+		-DCMAKE_INSTALL_PREFIX:PATH="$pkgdir/usr" \
+		..
+	cmake --build .
+}
+
+package() {
+	cd build
+	mkdir -p ${pkgdir}/usr
+	cmake --build . --target install
```

Comment:
i prefer `make install DESTDIR="$pkgdir"`

Reply:
Fixed. Thank you!